server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -81,6 +81,7 @@ private static IndexVersion def(int id, Version luceneVersion) {
     public static final IndexVersion V_8_9_0 = def(8_09_00_99, Version.LUCENE_9_7_0);
     public static final IndexVersion V_8_9_1 = def(8_09_01_99, Version.LUCENE_9_7_0);
     public static final IndexVersion V_8_10_0 = def(8_10_00_99, Version.LUCENE_9_7_0);
+    public static final IndexVersion V_8_13_0 = def(8_13_00_99, Version.LUCENE_9_9_0);
```

Comment:
The index version is now detached, see the definitions below. So instead of adding a `V_8_13_0` version you need to add one after the `UPGRADE_LUCENE_9_9`. Something like:
```
  public static final IndexVersion USER_DICT_CHECK_DUPLICATE = def(8_500_007, Version.LUCENE_9_9_0);
```

Reply:
@jimczi Thank you for the clarification. I appreciate your guidance regarding the detached state of IndexVersion. Although the information was present in the documentation, it appears I may have overlooked it due to the language barrier while using a translation tool. I will make the necessary adjustment by adding a new version after UPGRADE_LUCENE_9_9, as suggested